### ISSUE TITLE
Antivirus blocks Editor installer

### ISSUE THREAD

**Issue by narayb** [intechstudio/grid-editor#648]
Since v1.2.48 some users have reported that certain Anti-malware applications have blocked Editor installation either during an auto-update sequence or during manual download.

Malwarebytes (paid ver.) flagged Editor as an exploit and refused to let it run.
Avast quarantined the Editor installer.

**Comment by ohthatpete** [comment]
Yes, I'm blocked from installing it on my PC - Avast is having none of it!

**Comment by SukuWc** [comment]
Could be related to code signing
